fvtest/compilertest/CMakeLists.txt

```diff
@@ -122,4 +122,6 @@ target_link_libraries(compilertest
 
 set_property(TARGET compilertest PROPERTY FOLDER fvtest)
 
-add_test(NAME CompilerTest COMMAND compilertest --gtest_output=xml:${CMAKE_CURRENT_BINARY_DIR}/compilertest-results.xml)
+if (NOT OMR_HOST_OS STREQUAL "aix")
+	add_test(NAME CompilerTest COMMAND compilertest --gtest_output=xml:${CMAKE_CURRENT_BINARY_DIR}/compilertest-results.xml)
+endif()
```

Comment:
The test compiler tests also have issues with function descriptors. Not only do they not handle them when calling JITted code from the test harness, they also have tests for call opcodes which would not work if we added the existing function descriptor hack, since the linkage is not set up to be able to call via a function descriptor from within JITted code AFAIK.

Theoretically, this could be fixed but given that the test compiler tests are effectively deprecated at this point, I can't really justify putting the time into making them work.

Reply:
Ok, that is fair. Thanks for the explanation.